docs/guides/simulated_device_linux.md

```diff
@@ -88,15 +88,12 @@ interact with it using chip-tool
 1. Follow the instruction to build chip-tool in the
    [chip-tool readme](../../examples/chip-tool).
 
-2. Run this command to initiate the pairing.
+2. Run this command to commission.
     ```
-    ./out/debug/standalone/chip-tool pairing ethernet 0x654321 20202021 3842 [IP ADDRESS of App] 5542
+    ./out/debug/standalone/chip-tool pairing code MT:-24J0AFN00KA0648G00
```

Comment:
`pairing code $NODEID MT:....` ? it seems to be missing a nodeid.

Reply:
Yep, need to add that.  Good catch.